tests/e2e/deployment/deployment_test.go

```diff
@@ -76,8 +78,14 @@ var _ = Describe("Application deployment test in E2E scenario", func() {
 				}
 			}
 			utils.CheckPodRunningState(ctx.Cfg.ApiServer+AppHandler, podlist)
+			// End test timer
+			testTimer.End()
+			// Print result
+			testTimer.PrintResult()
```

Comment:
Do we need to store the time duration in somewhere?

Reply:
OK. Thanks

Reply:
Done, PTAL. Thanks